applovin_max/lib/applovin_max.dart

```diff
@@ -95,8 +96,8 @@ class AppLovinMAX {
     });
   }
 
-  static MaxAd _createAd(String adUnitId, dynamic arguments) {
     return MaxAd(
```

Comment:
Did you ever run this?

Reply:
Yeah - that was an incomplete plugin, just checked in the correct lines